MergerService/src/run.cs

```diff
@@ -51,153 +59,250 @@ private List<IData> BuildDataList(Source[] paths, int batchSize)
             }
         }
 
+        private List<KeyValuePair<string, string>> BuildTypeList()
+        {
+            var taskTypes = this._configurationManager.GetChildren("TASK", "types");
+
+            List<KeyValuePair<string, string>> values = new List<KeyValuePair<string, string>>();
+
+            foreach (var pair in taskTypes)
+            {
+                var jobType = pair.GetValue<string>("JobType");
+                var taskType = pair.GetValue<string>("taskType");
+                values.Add(new KeyValuePair<string, string>(jobType, taskType));
+            }
+
+            return values;
+        }
+
         public void Start()
         {
-            Stopwatch stopWatch = new Stopwatch();
-            TimeSpan ts;
+            var pollingTime = this._configurationManager.GetConfiguration<int>("TASK", "pollingTime");
```

Comment:
There is a log a few lines down.
In addition, why should i log the polling time...?

Reply:
It's more a general startup information - i won't insist on it but it will be clearer when reading the logs as to the fact the app was started with those values